README.md

```diff
@@ -1,26 +1,178 @@
+
+```Swift
+let videoRequest = Request<VIMVideo>(path: "/videos/45196609")
+```
+
+There are two critical takeaways here:
+- `<VIMVideo>`: the generic type of the model object.  This is what request will return in the `Response` if it's successful.
+- `path: "..."`: the API endpoint we want to call, you can see a whole host of other options at [Vimeo API Endpoints](https://developer.vimeo.com/api/endpoints)
+
+By declaring the expected model object type, we can ensure that the both the request and the parsing of the JSON is successful, and we can guarantee that our `Response` precisely matches this expectation.  All potential Vimeo models are already implemented in **VimeoNetworking**, and they're available for use in your own implementation.  Model classes are all named with the `VIM` suffix: `VIMUser`, `VIMChannel`, `VIMCategory`, and so on.
+
+### Handling Responses
+
+After we send that response, we'll get a `Result` enum back.  This could be either a `.Success` or a `.Failure` value.  `.Success` will contain a `Response` object, while `.Failure` will contain an `NSError`.  Switch between these two cases to handle whichever is encountered:
+
+```Swift
+vimeoClient.request(videoRequest) { result in 
+	switch result {
+	case .Success(let response: Response):
+		let video: VIMVideo = response.model
+		print("retrieved video: \(video)")
+	case .Failure(let error: NSError):
+		print("error retrieving video: \(error)"
+	}
+}
+```
+
+
+### Collections
+
+One neat **ProTip**: arrays of model objects are also valid generic types of `Request`.  In this case, the `model` property of your `Response` will be an array of parsed models of the expected type.
+
+```Swift
+let staffPickedVideosRequest = Request<[VIMVideo]>(path: "/channels/staffpicks/videos")
+
+vimeoClient.request(staffPickedVideosRequest) { result in 
+	switch result {
+	case .Success(let response: Response):
+		let videos: [VIMVideo] = response.model
+		for video in videos {
+			print("retrieved video: \(video)")
+		}
+	case .Failure(let error: NSError):
+		print("error retrieving videos: \(error)"
+	}
+}
+```
+
+
+## Last remarks
+
+With *all that* said, you now have a pretty solid understanding of what **VimeoNetworking** can do.  There's always more to explore, and we encourage you to play with the sample project, or dive right into the code and try it out yourself.  Most of our classes and functions are decently documented in the source files, so more detail on any topic can be found there.  If you still have questions or you're running into trouble, feel free to [file an issue](https://github.com/vimeo/VimeoNetworking/issues).  Better yet, if you fixed an issue or you have an improvement you'd like to share, send us a [pull request](https://github.com/vimeo/VimeoNetworking/pulls).  
```

Comment:
Can we also include info on:

- [ ] Requesting from cache vs network?
- [ ] Info about how to logout, about the accountDidChange notification?
- [ ] PATCH request with parameters? (is this overkill? Imo it's not...)